Quick start : throwing error Received error 500 from server for URI http://datahub-gms:8080/datasets

**Describe the bug**
Followed the Quick Start guides. Ingested sample data as well.
No data new data was available in the Datahub UI.

Throws following errors:

> datahub-frontend        | 09:33:14 [application-akka.actor.default-dispatcher-24] ERROR application - Failed in documents search.
> datahub-frontend        | com.linkedin.restli.client.RestLiResponseException: com.linkedin.restli.client.RestLiResponseException: Response status 500, serviceErrorMessage: INTERNAL SERVER ERROR
> datahub-frontend        | 	at com.linkedin.restli.internal.client.ExceptionUtil.wrapThrowable(ExceptionUtil.java:130)
> datahub-frontend        | 	at com.linkedin.restli.internal.client.ResponseFutureImpl.getResponseImpl(ResponseFutureImpl.java:130)
> .
> .
> datahub-frontend        | Caused by: com.linkedin.restli.client.RestLiResponseException: RestException{_response=RestResponse[headers={content-length=11045, Date=Thu, 26 Nov 2020 09:33:14 GMT, Server=Jetty(9.4.20.v20190813), X-RestLi-Error-Response=true, X-RestLi-Protocol-Version=2.0.0},cookies=[],status=500,entityLength=11045]}
> datahub-frontend        | 	at com.linkedin.restli.internal.client.ExceptionUtil.exceptionForThrowable(ExceptionUtil.java:102)
> datahub-frontend        | 	at com.linkedin.restli.client.RestLiCallbackAdapter.convertError(RestLiCallbackAdapter.java:66)
> datahub-frontend        | 	at com.linkedin.common.callback.CallbackAdapter.onError(CallbackAdapter.java:86)
> datahub-frontend        | 	at com.linkedin.r2.message.timing.TimingCallback.onError(TimingCallback.java:81)
> datahub-frontend        | 	at com.linkedin.r2.transport.common.bridge.client.TransportCallbackAdapter.onResponse(TransportCallbackAdapter.java:47)
> datahub-frontend        | 	at com.linkedin.r2.filter.transport.FilterChainClient.lambda$createWrappedClientTimingCallback$0(FilterChainClient.java:113)
> datahub-frontend        | 	at com.linkedin.r2.filter.transport.ResponseFilter.onRestError(ResponseFilter.java:79)
> datahub-frontend        | 	at com.linkedin.r2.filter.TimedRestFilter.onRestError(TimedRestFilter.java:92)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator$FilterChainRestIterator.doOnError(FilterChainIterator.java:166)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator$FilterChainRestIterator.doOnError(FilterChainIterator.java:132)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator.onError(FilterChainIterator.java:101)
> datahub-frontend        | 	at com.linkedin.r2.filter.TimedNextFilter.onError(TimedNextFilter.java:48)
> datahub-frontend        | 	at com.linkedin.r2.filter.message.rest.RestFilter.onRestError(RestFilter.java:84)
> datahub-frontend        | 	at com.linkedin.r2.filter.TimedRestFilter.onRestError(TimedRestFilter.java:92)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator$FilterChainRestIterator.doOnError(FilterChainIterator.java:166)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator$FilterChainRestIterator.doOnError(FilterChainIterator.java:132)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator.onError(FilterChainIterator.java:101)
> datahub-frontend        | 	at com.linkedin.r2.filter.TimedNextFilter.onError(TimedNextFilter.java:48)
> datahub-frontend        | 	at com.linkedin.r2.filter.message.rest.RestFilter.onRestError(RestFilter.java:84)
> datahub-frontend        | 	at com.linkedin.r2.filter.TimedRestFilter.onRestError(TimedRestFilter.java:92)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator$FilterChainRestIterator.doOnError(FilterChainIterator.java:166)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator$FilterChainRestIterator.doOnError(FilterChainIterator.java:132)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator.onError(FilterChainIterator.java:101)
> datahub-frontend        | 	at com.linkedin.r2.filter.TimedNextFilter.onError(TimedNextFilter.java:48)
> datahub-frontend        | 	at com.linkedin.r2.filter.message.rest.RestFilter.onRestError(RestFilter.java:84)
> datahub-frontend        | 	at com.linkedin.r2.filter.TimedRestFilter.onRestError(TimedRestFilter.java:92)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator$FilterChainRestIterator.doOnError(FilterChainIterator.java:166)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator$FilterChainRestIterator.doOnError(FilterChainIterator.java:132)
> datahub-frontend        | 	at com.linkedin.r2.filter.FilterChainIterator.onError(FilterChainIterator.java:101)
> datahub-frontend        | 	at com.linkedin.r2.filter.TimedNextFilter.onError(TimedNextFilter.java:48)
> datahub-frontend        | 	at com.linkedin.r2.filter.transport.ClientRequestFilter.lambda$createCallback$0(ClientRequestFilter.java:102)
> datahub-frontend        | 	at com.linkedin.r2.transport.http.common.HttpBridge$1.onResponse(HttpBridge.java:82)
> datahub-frontend        | 	at com.linkedin.r2.transport.http.client.rest.ExecutionCallback.lambda$onResponse$0(ExecutionCallback.java:64)
> datahub-frontend        | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
> datahub-frontend        | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
> datahub-frontend        | 	at java.lang.Thread.run(Thread.java:748)
> datahub-frontend        | Caused by: com.linkedin.r2.message.rest.RestException: Received error 500 from server for URI http://datahub-gms:8080/datasets
> datahub-frontend        | 	at com.linkedin.r2.transport.http.common.HttpBridge$1.onResponse(HttpBridge.java:76)
> datahub-frontend        | 	... 4 common frames omitted
> 
<img width="1221" alt="Screen Shot 2020-11-26 at 18 40 53" src="https://user-images.githubusercontent.com/70580659/100334501-38c8d900-3017-11eb-9183-7ebfd7523a1c.png">
<img width="1481" alt="Screen Shot 2020-11-26 at 18 41 25" src="https://user-images.githubusercontent.com/70580659/100334506-3a929c80-3017-11eb-87b5-25464028e366.png">
<img width="1262" alt="Screen Shot 2020-11-26 at 18 41 37" src="https://user-images.githubusercontent.com/70580659/100334512-3bc3c980-3017-11eb-87e3-baa40912ece2.png">




**To Reproduce**
Steps to reproduce the behavior:
1. clone the datahub repo
2. start ./docker/quickstart.sh
3. ingest sample data using /docker/ingestion/ingestion.sh
4. try to search for sample dataset

**Expected behavior**
sample dataset should be searchable 

**Screenshots**
If applicable, add screenshots to help explain your problem.

--- a/docker/elasticsearch-setup/dataset-index-config.json
+++ b/docker/elasticsearch-setup/dataset-index-config.json
@@ -190,8 +190,7 @@
             "keyword": {
               "type": "keyword"
             }
-          },
-
+          }
         },
         "hasOwners": {
           "type": "boolean"

